setup.cfg

```diff
@@ -0,0 +1,20 @@
+[metadata]
+name = dftbplus-ase
+version = 0.1
+author = DFTB+ developers group
+author_email = info@dftbplus.org
+url = https://github.com/dftbplus/dftbplus-ase
+description = Interfacing DFTB+ with the Atomic Simulation Environment
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+license = LGPL
```

Comment:
Use an SPDX identifier here (`LGPL-3.0-or-later`), see https://spdx.org/licenses/ for details.